core-parser/src/main/java/org/uniprot/core/parser/tsv/uniparc/UniParcEntryLightValueMapper.java

```diff
@@ -93,6 +95,13 @@ private Map<String, String> getSimpleAttributeValues(UniParcEntryLight entry, Li
                                     .map(LocalDate::toString)
                                     .orElse(""));
                     break;
+                case "common_taxons":
+                    map.put(
+                            UNIPARC_FIELDS.get(9),
+                            Optional.of(entry.getCommonTaxons())
+                                    .map(ct -> ct.stream().map(CommonOrganism::getCommonTaxon).collect(Collectors.joining("; ")))
+                                    .orElse(""));
+                    break;
```

Comment:
Too much cognitive load while reading the value collection for common_taxons.
Please extract it to a method getCommonTaxonsStr(commonTaxons)